### ISSUE TITLE
Clean application config

### ISSUE THREAD

**Issue by danmarcab** [MyMedsAndMe/spell#29]
The configuration for default serializer and transport looks like:
```elixir
config :spell,
  serializer: Spell.Serializer.JSON,
  transport: Spell.Transport.WebSocket
```
We want to change it to something less explicit like:
```elixir
config :spell,
  serializer: "json"
  transport: "websocket"
```
That way we could simplify the handling of this in `Mix.Tasks.Test.Integration` and `Crossbar`

**Comment by danmarcab** [comment]
Trying to use spell as a dependency I found out that if you don't define this config, spell crashes. Working on it

**Comment by danmarcab** [comment]
Fixed by #32
